test/bindings/python/test_utrecht.py

```diff
@@ -40,17 +40,17 @@ def tearDownClass(cls):
         del cls.actor
 
     def test_config(self):
-        config = get_config(self.tiles_path, self.extract_path)
+        config = get_config(self.extract_path, self.tiles_path)
```

Comment:
for some reason the python tests failed.. I couldn't see why exactly, but there were clearly a few long-standing bugs in the test code: `get_config()` expects the tar path first, then the tile_dir. I also added some code in the that function which protects against turning `""` to current working dir (default when initializing e.g. `Path("")`), so we can give it an empty path if necessary. that's a pretty annoying behavior of python's filesystem implementation that one can't detect if an empty string was used to initialize.. the c++ one would have `path.empty() == true` on a `std::filesystem::path path("")`.